src/App.jsx

```diff
@@ -28,13 +29,26 @@ function App() {
     if (selectedGenders.female && user.gender === "female") return true;
     return false;
    });
-
+   const ascendingEvent =() =>{
+    setIsAscending(!isAscending);
+  };
+  const sortedData = [...filteredData];
```

Comment:
You don't need to clone filteredData. `.filter` function will already create a new array from `myData`